lib/file-utilities.js

```diff
@@ -223,7 +223,7 @@ function streamAllLogs(config, callback) {
       return {
         t: data.__REALTIME_TIMESTAMP
       , line: data.MESSAGE
-      , f: data.CONTAINER_NAME || data.SYSLOG_IDENTIFIER || 'UNKNOWN_SYSTEMD_APP'
+      , f: data.CONTAINER_NAME || data._SYSTEMD_UNIT || data.SYSLOG_IDENTIFIER || 'UNKNOWN_SYSTEMD_APP'
```

Comment:
For the record, I have figured out to get this tested using journald within Docker, but it needs polished.  It was a pretty painful process.

Reply:
Gr8!